charts/lagoon-build-deploy/templates/deployment.yaml

```diff
@@ -47,6 +47,8 @@ spec:
         - /manager
         {{- if (or .Values.rootlessBuildPods .Values.extraArgs) }}
```

Comment:
I think this logic won't work anymore with `extraArgs` being empty. In that case there will be no arguments at all to `/manager`.

I think it will work if you just remove this outer `if` statement.

Reply:
🤦 ah yea. Will fix